app/tests/evaluation_tests/test_views.py

```diff
@@ -587,10 +587,26 @@ def test_hidden_phase_visible_for_admins_but_not_participants(client):
             reverse_kwargs={"challenge_short_name": ch.short_name, **kwargs},
             user=u,
         )
+
+        if view_name in [
+            "submission-list",
+            "submission-create",
+            "submission-detail",
+        ]:
+            icon = "upload"
+        else:
+            icon = "trophy"
+
         assert response.status_code == status
         if status == 200:
-            assert f"{visible_phase.title}</a>" in str(response.content)
-            assert f"{hidden_phase.title}</a>" not in str(response.content)
+            assert (
+                f'<i class="fa fa-{icon} fa-fw mr-1"></i>{visible_phase.title}'
+                in str(response.content)
+            )
+            assert (
+                f'<i class="fa fa-{icon} fa-fw mr-1"></i>{hidden_phase.title}'
+                not in str(response.content)
```

Comment:
Why are you checking for the icon here? With my earlier comment I meant checking for the icon you added (the eye icon), but I honestly think that just checking for the title would do here as well.

Reply:
Got it. I misinterpreted your comment. I will check for the title only.

Reply:
@ammar257ammar This feedback was never addressed. I updated the test to only check for the title.